packages/storybook/stories/ImageGallery/ImageGallery.stories.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ImageGallery as ImageGalleryComponent, ImageGalleryImageProps } from '@azure/communication-react';
+import { Title, Description, Heading, Source, Canvas } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+import { DetailedBetaBanner } from '../BetaBanners/DetailedBetaBanner';
+import { COMPONENT_FOLDER_PREFIX } from '../constants';
+import { controlsToAdd, hiddenControl } from '../controlsUtils';
+
+import { ImageGalleryExample } from './snippets/ImageGallery.snippet';
+const ImageGalleryExampleText = require('!!raw-loader!./snippets/ImageGallery.snippet.tsx').default;
+const importStatement = `import { ImageGallery } from '@azure/communication-react';`;
+
+const getDocs: () => JSX.Element = () => {
+  return (
+    <>
+      <Title>ImageGallery</Title>
+      <DetailedBetaBanner />
+      <Description>
+        Component to display image in a gallery. The gallery launches as a modal, it takes focus from the page or app
+        and require users to interact with them.
+      </Description>
+
+      <Heading>Importing</Heading>
+      <Source code={importStatement} />
+
+      <Heading>Example</Heading>
+      <Description>Component will render a fullscreen modal using a set image source.</Description>
+      <Canvas mdxSource={ImageGalleryExampleText}>
+        <ImageGalleryExample />
+      </Canvas>
+    </>
+  );
+};
+
+const ImageGalleryStory = (args): JSX.Element => {
+  // const [galleryImages, setGalleryImages] = useState<Array<ImageGalleryImageProps> | undefined>(undefined);
+  const galleryImage: ImageGalleryImageProps = {
+    title: 'Image',
+    saveAsName: 'images/inlineImageExample1.png',
+    imageUrl: 'images/inlineImageExample1.png'
+  };
+  const galleryImages = [galleryImage];
+  return (
+    <div style={{ width: '31.25rem' }}>
+      <ImageGalleryComponent
+        isOpen={args.disabled}
```

Comment:
Should this be enabled instead of disabled? And why don't we just use isOpen instead of giving a new name to it?

Reply:
`args.disabled` is the argument passed in from storybook. So we are not using a new name

Reply:
> `args.disabled` is the argument passed in from storybook. So we are not using a new name

I think the prop name in args are defined by us. It might be better to have isOpen={args.isOpen} or isOpen={args.displayImageGallery}
And correspondingly, in the argTypes section, we can have
  `argTypes: {
        isOpen: controlsToAdd.isOpen,
        ...
    }`
or
  `argTypes: {
        displayImageGallery: controlsToAdd.displayImageGallery,
        ...
    }`